community/pmc-tool/requirements.txt

```
@@ -0,0 +1,3 @@
+render==1.0.0
```

we don't need `render`, right? we do need requests:

```suggestion
requests
```